library/src/main/kotlin/com/chuckerteam/chucker/internal/ui/MainActivity.kt

```diff
@@ -136,12 +136,11 @@ internal class MainActivity :
     }
 
     private fun exportTransactions(fileName: String, block: suspend (List<HttpTransaction>) -> Sharable) {
+        val applicationContext = this.applicationContext
```

Comment:
Because a coroutine is being launched below that tries to access the Activity, we would be holding an instance.  By pulling out `applicationContext` into a local shadow `val`, we don't need to retain an instance of the `Activity`.